yarn.lock

```diff
@@ -69,6 +69,11 @@
   version "4.5.12"
   resolved "https://codeload.github.com/tinymce/tinymce-dist/tar.gz/3d007282ded510e32c638b2ed5a9ffe1f219d329"
 
+angular-sessionstorage@^1.1.5:
```

Comment:
Do we actually need this new dependency?

Reply:
Right now its used for the "link to" functionality in the indicators dossier. I'll look into a possible alternative.